src/sidebars/favourites/favourites.less

```diff
@@ -1,11 +1,13 @@
-﻿@favourite-height: 100px;
+﻿@favourite-height: 110px;
 @favourite-padding-left: 0px;
-@favourite-padding-top: 5px;
-@favourite-padding-right: 4px;
-@favourite-padding-bottom: 10px;
+@favourite-padding-top: 10px;
+// Padding right is declared in the variables file.
```

Comment:
Although moving it out is justified, I'm not too sure on having only `favourite-padding-right` moved out. it doesn't seem too consistent to me. Perhaps have a new variable, which is then assigned to `@favourite-padding-right` in this file?